.gitignore

```diff
@@ -266,3 +266,5 @@ RetrospectiveExtension.FrontEnd/vss-extension.dev.json
 .env
 allowed_origins.json
 dev_certs.json
+
```

Comment:
We should include anything we provide templates for here.

Reply:
done added templates in the same place for consistency